src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -563,15 +565,17 @@ public void ActiveSolutionChanged_GitEventsMonitorRefreshInvoked()
             }
         }
 
-        private static ActiveSolutionBoundTracker CreateTestSubject(IHost host,
+        private ActiveSolutionBoundTracker CreateTestSubject(IHost host,
             IActiveSolutionTracker solutionTracker,
             IConfigurationProvider configurationProvider,
             ILogger logger = null,
-            IBoundSolutionGitMonitor gitEvents = null)
+            IBoundSolutionGitMonitor gitEvents = null,
+            IServiceProvider serviceProvider = null)
         {
             logger ??= new TestLogger(logToConsole: true);
             gitEvents ??= Mock.Of<IBoundSolutionGitMonitor>();
-            return new ActiveSolutionBoundTracker(host, solutionTracker, logger, gitEvents, configurationProvider);
+            serviceProvider ??= this.serviceProvider;
```

Comment:
This test class still has a `[TestInitialize]` that sets up most of the mocks used. Removing the `[TestInitialize]` would have created too much churn. Instead, I changed this method to allow tests to supply a custom `IServiceProvider` if required. By default, the one configured in `TestInitialize` is used.